tests/test_flow/test_persistence.py

```diff
@@ -821,3 +821,58 @@ def y():
 
         flow = builder.build()
         assert flow.get('y') == 1
+
+
+def test_volatile_ancestor(builder):
```

Comment:
Any idea how hard it would be to incorporate volatile entities into `test_persistence_fuzz.py`?

Reply:
it wasn't as easy as I hoped but done